docs/topics/proc_deploying-the-health-check-booster-using-the-oc-cli-client.adoc

```diff
@@ -79,7 +80,7 @@ The number in the middle will increase with each new build.
 The letters at the end are generated when the pod is created.
 
 
-. Once your booster is deployed and started, determine its route.
+. Once your example is deployed and started, determine its route.
```

Comment:
Once your example is deployed and started,--> Once your example application is deployed and started,